staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -2259,6 +2260,130 @@ func TestCELMaxRecursionDepth(t *testing.T) {
 	}
 }
 
+func TestMessageExpression(t *testing.T) {
```

Comment:
I was expecting to see a test that produced a runtime error.

Reply:
`runtime cost preserved if messageExpression fails during evaluation` maybe? That one fails during evaluation - was there another sort of runtime error you were looking for?

Reply:
Ah i missed that. LGTM